source/index.html

```diff
@@ -2,41 +2,51 @@
 <html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Мишка - милые штуки ручной работы для дома</title>
     <link rel="stylesheet" href="css/style.css">
   </head>
   <body class="page__body">
     <header class="main-header">
+      <div class="container">
+
+        <nav class="main-nav main-nav--closed main-nav--nojs">
+          <div class="main-header__wrapper">
+            <a class="main-header__logo" href="index.html">
+              <img class="main-header__logo-image" src="img/mishka-logo-mobile.svg" width="86" height="35" alt="Логотип магазина товаров ручной работы «Мишка»">
+            </a>
+            <button class="main-nav__toggle" type="button">
+            <span class="visually-hidden">Открыть меню</span>
+            </button>
+          </div>
+            <ul class="main-nav__list site-list">
+              <li class="main-nav__item site-list__item">
+                <a class="main-nav__link" href="catalog.html">Каталог товаров</a>
+              </li>
+              <li class="main-nav__item site-list__item">
+                <a class="main-nav__link" href="form.html">Вязание на заказ</a>
+              </li>
+            </ul>
+            <ul class="main-nav__list user-list">
+              <li class="main-nav__item user-list__item">
+                <a class="main-nav__link" href="#"><span>Поиск</span></a>
+              </li>
+              <li class="main-nav__item user-list__item">
+                <a class="main-nav__link" href="#">Корзина: <span>пока пуста</span></a>
+              </li>
+            </ul>
+            <ul class="main-nav__list main-nav__list--desktop">
+              <li class="main-nav__item main-nav__item--desktop">
+                <a class="main-nav__link" href="#"><span>Новые поступления</span></a>
+              </li>
+              <li class="main-nav__item main-nav__item--desktop">
+                <a class="main-nav__link" href="#">Распродажа</span></a>
```

Comment:
Потерян открывающий тег `<span>`:
```suggestion
                <a class="main-nav__link" href="#">Распродажа</span></a>
```
Но по хорошему здесь и в ссылке "Новые поступления" `span` не нужен и является лишним элементом разметки. Есть два момента, когда текст в ссылке (кнопке) нужно обернуть в `span`:
1. Текста в явном виде нет, вместо него иконка. Как в ссылке "Поиск". Поясняющий текст скрываем при помощи `visually-hidden`.
2. В тексте ссылки что-то изменяется динамически и ту часть, которая зависит от условий оборачиваем в `span`. Как в ссылке "Корзина" вы правильно обернули текст "пока пуста", потому что при наличии товаров в корзине здесь будет из количество.